pkg/apis/pipeline/v1alpha1/run_types.go

```diff
@@ -60,6 +60,10 @@ type RunSpec struct {
 	// +optional
 	Status RunSpecStatus `json:"status,omitempty"`
 
+	// Status message for cancellation.
+	// +optional
+	StatusMessage RunSpecStatusMessage `json:"statusMessage,omitempty"`
```

Comment:
could you add some info to docs/runs.md about how custom task controllers should support this message?

Reply:
Added some info in runs.md on how statusMessage can be used by the custom controller developer.